### DISCUSSION TITLE
Forked public versions - FINNoco [Jan 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4939] [Discord Threads]
> 🙋‍♂️ **Jonathan Leung**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Jakob from bunch**  
#### hi jonathan, check out FINNoco version  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054799589401710612)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
